src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -81,111 +82,52 @@ private static Version GetICUVersion()
                 }
             }
-            else 
-            {
-                string fileName = null;
-                if (File.Exists("/etc/redhat-release"))
-                    fileName = "/etc/redhat-release";
-
-                if (fileName == null && File.Exists("/etc/system-release"))
-                    fileName = "/etc/system-release";
-                
-                if (fileName != null)
-                {
-                    // Parse the format like the following line:
-                    // Red Hat Enterprise Linux Server release 7.3 (Maipo)
-                    using (StreamReader file = new StreamReader(fileName))
-                    {
-                        string line = file.ReadLine();
-                        if (!String.IsNullOrEmpty(line))
-                        {
-                            if (line.StartsWith("Red Hat Enterprise Linux", StringComparison.OrdinalIgnoreCase))
-                            {
-                                ret.Id = "rhel";
-                            }
-                            else if (line.StartsWith("Centos", StringComparison.OrdinalIgnoreCase))
-                            {
-                                ret.Id = "centos";
-                            }
-                            else if (line.StartsWith("Red Hat", StringComparison.OrdinalIgnoreCase))
-                            {
-                                ret.Id = "rhl";
-                            }
-                            else 
-                            {
-                                // automatically generate the distro label
-                                string [] words = line.Split(' ');
-                                StringBuilder sb = new StringBuilder();
 
-                                foreach (string word in words)
-                                {
-                                    if (word.Length > 0)
-                                    {
-                                        if (Char.IsNumber(word[0]) || 
-                                            word.Equals("release", StringComparison.OrdinalIgnoreCase) ||
-                                            word.Equals("server", StringComparison.OrdinalIgnoreCase))
-                                            {
-                                                break;
-                                            }
-                                        sb.Append(Char.ToLower(word[0]));
-                                    }
-                                }
-                                ret.Id = sb.ToString();
-                            }
+            return false;
+        }
 
-                            int i = 0;
-                            while (i < line.Length && !Char.IsNumber(line[i])) // stop at first number
-                                i++;
+        private static bool VersionEqual(string expectedVersionId, string actualVersionId)
+        {
+            if (expectedVersionId == null)
```

Comment:
Would it make sense to parse these into `Version` structs and use the comparisons that come with it?